### PR TITLE
Setup app for "My first react project"

### PR DESCRIPTION
**Pull request by lmunozm1702**
### This PR includes:

- Configure git repository using Gitflow (master <- development <- feature branches)
- Create react app using npx create-react-app my-first-react
- Configure linters (https://github.com/microverseinc/linters-config/tree/master/react-redux).

The application runs in the browser without errors:

image

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Linters
```
Comment (rustam-tolipov) [inline .github/workflows/linters.yml:1]: Screenshot 2022-11-28 at 21 41 41

**Review by rustam-tolipov — COMMENTED** [review]
Hi @lmunozm1702, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter errors. To make your code reviewable all linters should be green. Kindly fix them for the next shot 😊. Happy coding :)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (lmunozm1702) [reply .github/workflows/linters.yml:1]: HI @Rustamxon7, the errors are already solved... Regards.

**Review by tugsem — COMMENTED** [review]
## Invalid Code Review Request ❌ 

Hi @lmunozm1702 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h2>You have submitted a project, that doen't have readme template provided by Microverse.</h2>

- [ ] All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hi @lmunozm1702 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ Correct use of Gitflow
✔️ Good readme file
✔️ Descriptive pr messages
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**